index.html

```diff
@@ -7,92 +7,207 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Figma Project</title>
   <link rel="stylesheet" href="./style.css">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap" rel="stylesheet">
+</head>
 
 <body>
-  <div class="first_page">
-    <header class="header flex_container">
-      <div class="logo">ReMotely</div>
-      <nav class="nav flex_container">
-        <div class="flex_container"><a href="#">book</a><a href="#">explore</a><a href="#">about</a></div>
-        <div><a href="#">sign in</a></div>
-      </nav>
-    </header>
-
-
-    <div class="catch_phrase">
-      <h1>Work from a Cabin near Helsinki</h1>
-      <button class="btn" type="submit">Explore</button>
-      <img class="main_photo" src="switzphoto.jpg" alt="chatka ze Szwajcarii">
-    </div>
-  </div>
 
 
-  <div class="second_page_container">
-    <div class="paragraph">
-      <p>We offer a handpicked range of cabins for nature lovers in Europeto live and work from for desired time.</p>
-    </div>
-    <div class="pros_container">
-      <div class="photo_description_container">
-        <div class="pros_photo"><img class="pros_pic" src="forest.png" alt="zdjęcie zalet"></div>
-        <div class="paragraph_container">
-          <header class="paragraph_header">Connection</header>
-          <p>We provide a good internet connection in our remotest cabins</p>
+
+<div class="whole_page_wrapper">
+
+
+  
+  <header class="header section flex_container">
+    <div class="logo">ReMotely</div>
+    <nav class="nav flex_container">
+      <div class="link_flex_container"><a class="links" href="#">book</a><a class="links" href="#">explore</a>
+        <a class="links" href="#">about</a>
+      </div>
+      <div class="sign_container"><a class="sign" href="#">sign in</a></div>
+    </nav>
+  </header>
```

Comment:
this was supposed to follow the scroll (be always visible on top)

Reply:
fixed

Reply:
@sinkinshadow good. Now make it appear in one row.

Reply:
fixed